politeiawww/api/v1/v1.go

```diff
@@ -184,12 +182,13 @@ type CensorshipRecord struct {
 
 // ProposalRecord is an entire proposal and it's content.
 type ProposalRecord struct {
-	Name      string      `json:"name"`      // Suggested short proposal name
-	Status    PropStatusT `json:"status"`    // Current status of proposal
-	Timestamp int64       `json:"timestamp"` // Last update of proposal
-	PublicKey string      `json:"publickey"` // Key used for signature.
-	Signature string      `json:"signature"` // Signature of merkle root
-	Files     []File      `json:"files"`     // Files that make up the proposal
+	Name        string      `json:"name"`                  // Suggested short proposal name
+	Status      PropStatusT `json:"status"`                // Current status of proposal
+	Timestamp   int64       `json:"timestamp"`             // Last update of proposal
+	PublicKey   string      `json:"publickey"`             // Key used for signature.
+	Signature   string      `json:"signature"`             // Signature of merkle root
+	Files       []File      `json:"files"`                 // Files that make up the proposal
+	NumComments *uint       `json:"numcomments,omitempty"` // Proposal's comment count
```

Comment:
Why is this a pointer? 0 is a fine number. And is this the right location in the structure where this goes? Let's move it under Timestamp.

Reply:
It's a pointer because the json marshaller will not include it if nil > field will not be included for unvetted proposals.